enterprise/admin/views.py

```diff
@@ -213,7 +219,7 @@ def _handle_bulk_upload(cls, enterprise_customer, manage_learners_form, request)
                     list_of_emails=", ".join(duplicate_emails)
                 )
             )
-        return list(emails) + already_linked_emails
+        return list(emails) + [email for email, __ in already_linked_emails]
```

Comment:
Good call! Also, looking into this comment made me realize a bug in the code that I need to fix, so thanks!